### PR TITLE
Add dir support to overlayfs_mount

### PR DESCRIPTION
**Pull request by ericpe**
allow overlayfs_mount to support using directories as lowerdirs

### CODE REVIEW

**Comment by ericpe** [comment]
forgot to pull master, had to rebase

File: share/overlayfs.sh [hunk 1]
```diff
@@ -134,7 +134,13 @@ overlayfs_mount()
             sources+=( "${src}" )
 
             ln -s "${src}" "${lower_src}"
-            archive_mount "${src}" "${lower}"
+
+            # but, what if an image isn't an image and is actually a directory
```
Comment (thecodenomad) [inline share/overlayfs.sh:138]: \m/ Good call! :D

**Review by marshall-mcmullen — APPROVED** [review]
All the tests passed. This looks like a really good change to me. Approved!

**Comment by thecodenomad** [comment]
w00t!

**Pull request merged**